src/content/docs/apm/agents/ruby-agent/getting-started/migration-9x-guide.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Ruby agent 8.x to 7.x migration guide
+tags:
+  - Agents
+  - Ruby agent
+  - Getting started
+redirects:
+  - /docs/agents/ruby-agent/getting-started/migration-9x-guide
+---
+
+## Summary [#summary]
+
+This guide covers the major changes between the 8.x and 9.x series of the Ruby agent, issues that may be encountered while upgrading, and how to successfully migrate to version 9.x.
+
+The main changes include:
+
+* [Removed all deprecated configuration options](#removed_config)
+* [Dropped support for Ruby 2.2 and 2.3](#ruby_2223)
+* [Enabled Thread tracing](#thread_tracing)
+* [Removed deprecated instrumntation for gems](#removed_gem_instrumentation)
+* [Updated API method `set_transaction_name`](#set_transaction_name)
+* [Removed deprecated API methods](#removed_api_methods)
```

Comment:
@kaylareopelle we organized this based on what we thought could potentially have the most impact on customers. I agree removing deprecated content is unsurprising, but also that it could have a more breaking impact. Thoughts?

Reply:
My apologies! I should have asked if it was already prioritized. Seeing this was previously considered, please disregard my comment.

Reply:
I'm not sure why I can't mark this as resolved, but it should be :)